### ISSUE TITLE
DNS pointer message shows up even though DNS is already in place and cert is generated

### ISSUE THREAD

**Issue by raphaeltm** [DefangLabs/defang#357]
I keep getting this message when I run `defang compose up`:

```
** Please point the DNS record for hasura.hasura-rds-demo-1.goec.dev as a CNAME to Defang-hasura-rds-beta-alb-457580578.us-east-1.elb.amazonaws.com and access http://hasura.hasura-rds-demo-1.goec.dev to trigger the certificate issuance.
```

But I already have the DNS records in place, and the cert has been generated, so it should be all good.

**Comment by edwardrf** [comment]
This will be fixed as part of the up coming [PR #658](https://github.com/defang-io/defang-mvp/pull/658/files#diff-c643e69cd595b1e7a603950ecb86fac2ed950494a7bac5530c459bebbd2b47baL240-L245)

**Comment by edwardrf** [comment]
Fixed by [https://github.com/DefangLabs/defang-mvp/pull/658](https://github.com/DefangLabs/defang-mvp/pull/658/files#diff-c643e69cd595b1e7a603950ecb86fac2ed950494a7bac5530c459bebbd2b47baL235-L240)
